component-server-parent/component-server/src/main/java/org/talend/sdk/component/server/front/base/internal/RequestKey.java

```diff
@@ -28,10 +28,13 @@ public class RequestKey {
 
     private final int cacheHash;
 
-    public RequestKey(final Locale locale, final boolean includeIconContent, final String query) {
+    private final String theme;
+
+    public RequestKey(final Locale locale, final boolean includeIconContent, final String query, final String theme) {
         this.locale = locale;
         this.includeIconContent = includeIconContent;
         this.query = query;
+        this.theme = theme;
         this.cacheHash = Objects.hash(locale, includeIconContent, query);
```

Comment:
You didn't change hashcode calculation, is it expected to not include it into hash, but check it in "equals"?

Reply:
Oh right, good catch!